resdk/query.py

```diff
@@ -241,13 +237,31 @@ def _fetch(self):
 
         # Extract data from paginated response
         if isinstance(items, dict) and 'results' in items:
+            self._count = items['count']
             items = items['results']
 
-        self._cache = [self._populate_resource(data, **resource_inputs) for data in items]
+        self._cache = [self._populate_resource(data, resolwe=self.resolwe) for data in items]
```

Comment:
The `self._populate_resource` and `self.resolwe` are both members of this class. Why do you have to pass a class attribute to a class method? Couldn't you simply access `self.resolwe` in `_populate_resource`?